openapi-ts version 0.53.3 breaks esm build

### Description

I have a file `generate-types.js` to run openapi-ts with this content

```js
import { createClient } from '@hey-api/openapi-ts';
import fs from 'node:fs';

const outputDir = 'src/data/types';

fs.rmSync(outputDir, { recursive: true, force: true });

await createClient({
  client: 'legacy/fetch',
  input: 'input.yaml',
  output: outputDir,
});
```

This ran just fine until version 0.53.2, but it seems the dual build thing in 0.53.3 broke this way of using openapi-ts.

```
> node generate-types.js

node:internal/modules/run_main:129
    triggerUncaughtException(
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/node_modules/handlebars/runtime' imported from /node_modules/@hey-api/openapi-ts/dist/index.js
Did you mean to import "handlebars/runtime.js"?
    at finalizeResolution (node:internal/modules/esm/resolve:265:11)
    at moduleResolve (node:internal/modules/esm/resolve:933:10)
    at defaultResolve (node:internal/modules/esm/resolve:1169:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:227:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:87:39)
    at link (node:internal/modules/esm/module_job:86:36) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///node_modules/handlebars/runtime'
}

Node.js v20.16.0
```

@mrlubos you made the commit, am I doing something wrong, or is this a bug?

Btw. if I change my script to `.cjs` and imports to require statements, it works just fine again.

### Reproducible example or configuration

_No response_

### OpenAPI specification (optional)

_No response_

### System information (optional)

_No response_

--- a/.changeset/big-seals-count.md
+++ b/.changeset/big-seals-count.md
@@ -1,0 +1,5 @@
+---
+'@hey-api/openapi-ts': patch
+---
+
+fix: import handlebars instead of runtime

--- a/packages/openapi-ts/src/utils/__tests__/handlebars.spec.ts
+++ b/packages/openapi-ts/src/utils/__tests__/handlebars.spec.ts
@@ -1,4 +1,4 @@
-import Handlebars from 'handlebars/runtime';
+import Handlebars from 'handlebars';
 import { describe, expect, it } from 'vitest';
 
 import { setConfig } from '../config';

--- a/packages/openapi-ts/src/utils/handlebars.ts
+++ b/packages/openapi-ts/src/utils/handlebars.ts
@@ -1,4 +1,4 @@
-import Handlebars from 'handlebars/runtime';
+import Handlebars from 'handlebars';
 
 // @ts-ignore
 import templateClient from '../legacy/handlebars/compiled/client.js';

